File: platform/sdk/nodejs/examples/cloud/cloud-030_dbhc_verify_cli_ok.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("dbhc-bucket");
client.dbhcAppend1kBlocks("dbhc-bucket", "chain.log", 3);
const result = client.dbhcVerifyCliOk("dbhc-bucket", "chain.log");
if (!result.verified) process.exit(1);
console.log("XJ-OK: cloud-030_dbhc_verify_cli_ok");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-030_dbhc_verify_cli_ok.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("dbhc-bucket")
client.dbhcAppend1kBlocks("dbhc-bucket", "chain.log", 3)
result = client.dbhcVerifyCliOk("dbhc-bucket", "chain.log")
if not result["verified"]:
    sys.exit(1)
print("XJ-OK: cloud-030_dbhc_verify_cli_ok")
sys.exit(0)
